### ISSUE TITLE
Global header navigation items :focus state not visible in Windows High Contrast mode

### ISSUE THREAD

**Issue by joedolson** [WordPress/wporg-mu-plugins#648]
In Windows High Contrast mode, the only characteristics of a :focus state that are visible are `outline` or `border`, which are both set to empty values on the global header. 

Core uses a global `outline: 2px solid transparent;`, which is invisible in standard modes, but becomes visible in high contrast mode. 

See https://meta.trac.wordpress.org/ticket/4111
